app/src/main/java/sdp/moneyrun/menu/NewGameImplementation.java

```diff
@@ -110,27 +113,45 @@ public void postNewGame(String name, int maxPlayerCount) {
         // Grant permissions if necessary
         requestLocationPermissions(requestPermissionsLauncher);
 
-        fusedLocationClient.getLastLocation()
-                .addOnSuccessListener(activity, location -> {
-                    // Got last known location. In some rare situations this can be null
-                    // In this case, the game cannot be instanciated
-                    if (location == null) {
-                        Log.e("location", "Error getting location");
-                    }
-
-                    // Build new game given fields filled by user
-                    List<Riddle> riddles = new ArrayList<>();
-                    List<Coin> coins = new ArrayList<>();
+        // Build new game given fields filled by user
+        List<Riddle> riddles = new ArrayList<>();
+        List<Coin> coins = new ArrayList<>();
+        //no node is created in the DB for an empty list. need to fill it with something
+        coins.add(new Coin(1,1,99));
+
```

Comment:
Do you remove this coin later on?

Reply:
No, you are right that it wasn't the best way of working around the problem. I have added a condition in GameDatabaseProxy, so that if the Coins node is null, it will create an empty arrayList instead